server/views/pages/overview.njk

```diff
@@ -8,9 +8,13 @@
 {% set errorMessage = "Data unavailable - try again later or contact administrator if problem persists" %}
 {% set addAppointmentEnabled = features.ADD_APPOINTMENTS | getFeatureFlag %}
 {% set resettlementAssessmentEnabled = features.RESETTLEMENT_ASSESSMENT | getFeatureFlag %}
+{% set tasksViewEnabled = features.TASKS_VIEW | getFeatureFlag %}
 {% block content %}
   <div class="govuk-grid-row govuk-!-padding-top-4 govuk-!-padding-bottom-4">
     {{ titleAndIntro(pathway) }}
+     <div class="govuk-grid-column-one-third">
```

Comment:
Should there be an if statement here to only render if the flag is set?

Reply:
oh yes sorry I forgot